dask/dataframe/core.py

```diff
@@ -2811,7 +2811,14 @@ def nunique(self, split_every=None):
         return self.drop_duplicates(split_every=split_every).count()
 
     @derived_from(pd.Series)
-    def value_counts(self, split_every=None, split_out=1):
+    def value_counts(
+        self, split_every=None, split_out=1, sort=None, ascending=False, dropna=None
```

Comment:
Does `sort` depend on the version of pandas, or can we make it True to match pandas behavior?

`dropna` being None by default makes sense, since it's version dependent.

Reply:
And I don't know what our API restrictions are on changing keyword order, but we should probably have all the pandas kwargs first, then our `split_every`, `split_out` kwargs at the end.

Reply:
My thinking with having `sort=None` is that it is a potentially expensive operation so I didn't want it to be on by default. I agree that  the pandas kwargs should go first and I doubt anyone is passing split_ever an split_out in positionally, so I'll just change the order.

Reply:
Hmm on `sort`, is that only called on the reduced output, so hopefully it's not too expensive?

Reply:
Yeah it is only on reduced output. I can make it True by default if you prefer. No strong opinion, just trying to be cautious.

Reply:
I just made it True by default.

Reply:
Actually I take it back. Sorting when there are multiple out partitions isn't guaranteed to work, so it is better to leave it off by default and let people turn it on if they need it. See https://github.com/dask/dask/pull/5880#discussion_r377208304